src/components/v2/appDetails/AppDetails.component.tsx

```diff
@@ -55,55 +44,84 @@ const AppDetailsComponent = ({
         (event) => setStreamData(JSON.parse(event.data)),
     );
 
+    return (
+        <div style={{ display: 'flex', flexDirection: 'column', flex: 1 }}>
+            <div>
+                <EnvironmentSelectorComponent />
+                <EnvironmentStatusComponent appStreamData={streamData}/>
+            </div>
+
+            <SyncErrorComponent appStreamData={streamData} />
+            <AppLevelExternalLinks helmAppDetails={appDetails} externalLinks={externalLinks} monitoringTools={monitoringTools} />
+            <NodeTreeDetailTab appDetails={appDetails} externalLinks={externalLinks} monitoringTools={monitoringTools} />
+            
+        </div>
+    );
+};
+
+export function NodeTreeDetailTab({
```

Comment:
Can we isolate this function as an individual component?

Reply:
done